test/README.md

```diff
@@ -83,8 +83,20 @@ Makefile targets
     test_results/compilable/json.d.out      runs an individual test
                                             (run log of the test is stored)
 
-In-test variables
+Test Configuration
 -----------------
+All tests defined within `.d` source files may be configured using the settings listed below, i.e.
+
+mytest.d
+```
+/*
+REQUIRED_ARGS: -version=Foo
+EXTRA_SOURCES: imports/foo.d
+*/
+// ... test code ...
+```
```

Comment:
Three idea that come to my mind:
- mention difference between `extra-files` and `imports` (i.e. `imports` is automatically added to the imports path)
- mention `TEST_OUTPUT` as it's probably the most commonly used option
- `EXTRA_SOURCES` is one of the things that gets "abused" easily and maybe we shouldn't recommend (i.e. if the file is in `imports`, you can simply `import` it, it's only required if you explicitly need the file as DMD argument and even then is your `-i` easier. So this leaves it useful only to e.g. ddoc files)

Reply:
Actually, `imports` is not added to the import paths

Reply:
Importing modules from the `imports` directory ONLY works if your import matches the filename, i.e.
```
--- test.d
version(WontWork)
    import imports.foo;
else
    import foo;
--- imports/foo.d
module foo;
```